tailwind.config.js

```diff
@@ -0,0 +1,123 @@
+module.exports = {
+  theme: {
+    borderColor: theme => ({
+      1: theme("colors.border.1"),
+      2: theme("colors.border.2"),
+      3: theme("colors.border.3"),
+      4: theme("colors.border.4"),
+      5: theme("colors.border.5"),
+    }),
+    borderWidth: {
+      0: "0",
+      1: "1px",
+      3: "3px"
+    },
+    borderRadius: {
+      none: '0',
+      1: "var(--calcite-border-radius)",
+      full: '100%',
+    },
+    boxShadow: {
+      1: {
+        default: "0 0 8px 0 rgba(0, 0, 0, 0.08)",
+        hover: "0 0 16px 0 rgba(0, 0, 0, 0.08)",
+        press: "0 0 8px 0 rgba(0, 0, 0, 0.16)"
+      },
+      2: {
+        default: "0 0 16px 0 rgba(0, 0, 0, 0.16)",
+        hover: "0 0 32px 0 rgba(0, 0, 0, 0.16)",
+        press: "0 0 16px 0 rgba(0, 0, 0, 0.32)",
+      }
+    },
+    colors: {
+      blue: {
+        1: "var(--calcite-ui-blue-1)",
+        2: "var(--calcite-ui-blue-2)",
+        3: "var(--calcite-ui-blue-3)",
+      },
+      green: {
+        1: "var(--calcite-ui-green-1)",
+        2: "var(--calcite-ui-green-2)",
+        3: "var(--calcite-ui-green-3)",
+      },
+      yellow: {
+        1: "var(--calcite-ui-yellow-1)",
+        2: "var(--calcite-ui-yellow-2)",
+        3: "var(--calcite-ui-yellow-3)",
+      },
+      red: {
+        1: "var(--calcite-ui-red-1)",
+        2: "var(--calcite-ui-red-2)",
+        3: "var(--calcite-ui-red-3)",
+      },
+      background: "var(--calcite-ui-background)",
+      foreground: {
+        1: "var(--calcite-ui-foreground-1)",
+        2: "var(--calcite-ui-foreground-2)",
+        3: "var(--calcite-ui-foreground-3)",
+      },
```

Comment:
I suspect we'll want to iterate on the colors (gray comes to mind) but this seems like a nicely constrained start. 

For our part, we have not been in the habit of using semantic color names other than [`brand-` and `alert-`]. This is for no good reason either, we essentially just lifted the tailwind naming scheme; so I will look forwarded to see how this plays out in our apps.

Reply:
We've discussed aliasing these before, with theming it does seem a bit weird that blue === brand color, but we've never aligned around nomenclature for red = alert, yellow = warning, green = success, blue = info, etc. We have components that have a color = prop, so if this were to change we'd want to update that to be something like status / state = error vs. color = red.

Would be happy to have that larger conversation again since we would want that to be updated before a 1.0

Reply:
:+1: Sounds like something we should put on the menu.